src/Carbon/Carbon.php

```diff
@@ -524,6 +553,9 @@ public function __get($name)
             case $name === 'quarter':
                 return (int) ceil($this->month / 3);
 
+            case $name === 'milliSeconds':
+                return (int) ceil($this->micro / static::MICROSECONDS_PER_MILLISECOND);
```

Comment:
As a Carbon user, I'd rather see a way to get the fractions of the second, however many decimal places it may be.  I don't want to be limited to `milliSeconds` (Is the capitalization intentional?), microseconds, etc.  Better yet, it's a part of the seconds, so return seconds as a floating point number.